fabric-common/lib/IdentityContext.js

```diff
@@ -84,11 +84,11 @@ const IdentityContext = class {
 	 * @param {Buffer} payload - The payload bytes that require a signature
 	 * @return {Buffer} - The signature in bytes
 	 */
-	sign(payload = checkParameter('payload')) {
+	async sign(payload = checkParameter('payload')) {
```

Comment:
JSDoc and type definitions also need to be updated to reflect this change

Reply:
The type definitions in `fabric-common/types/index.d.ts` also need to be changed to match the code.

Reply:
If the function is async then you could also include the `@async` keyword in the JSDoc